Makefile

```diff
@@ -72,7 +72,7 @@ test-unit: manifests generate fmt vet envtest ## Run unit tests.
 
 .PHONY: test-integration
 test-integration: manifests generate fmt vet envtest ## Run integration tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -tags=integration -coverprofile cover-integration.out -ginkgo.v -v -timeout 0
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./test/... -tags=integration -coverprofile cover-integration.out -ginkgo.v -v -timeout 0
```

Comment:
Might be better to just use ginkgo command here, see [e2e:](https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/Makefile#L83)

```
KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" $(GINKGO) -tags=integration -v ./test/integration
```

You'll also need to change the e2e target to run the specs in the e2e dir only:

```
$(GINKGO) -tags=e2e -v ./test/e2e
```